packages/fa-icon-chooser/src/utils/utils.ts

```diff
@@ -61,7 +61,7 @@ export interface IconDefinition extends IconLookup {
 
 export type IconChooserResult = IconLookup | IconDefinition;
 
-const viewBoxRe = /viewBox="0 0 ([0-9]+) ([0-9]+)"/;
+const viewBoxRe = /viewBox="([0-9]+[\s,]+[0-9]+[\s,]+[0-9]+[\s,]+[0-9]+)"/;
```

Comment:
I see that you've changed the viewBox regular expression here, generalizing it to match all kinds of valid viewBox syntax.

However, it's been generalized to cover cases that never occur in the SVGs that will be parsed. The only ones being parsed are the ones that are fetched from our Kits CDN, and we tightly control the format in which they are written. So I could make the case that if the more strict viewBox regex fails to match, then that represents a programming error elsewhere in our system.

Is there a case or concern that you've found that requires a more generalized regular expression here?